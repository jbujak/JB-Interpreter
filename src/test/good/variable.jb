int main() {
	int x = 1;
	string y = "a";
	bool z = true;

	print(int_to_string(x));
	print(y);
	print(bool_to_string(z));

	x++;
	print(int_to_string(x));
	x--;
	print(int_to_string(x));

	x += 5;
	print(int_to_string(x));
	x -= 5;
	print(int_to_string(x));
	x *= 4;
	print(int_to_string(x));
	x /= 4;
	print(int_to_string(x));

	x = 100;
	print(int_to_string(x));
}
