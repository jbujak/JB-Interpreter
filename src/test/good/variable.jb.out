1
a
true
2
1
6
1
4
1
100
